Node/angular-karma/karma.conf.js

```diff
@@ -0,0 +1,49 @@
+// Karma configuration file, see link for more information
+// https://karma-runner.github.io/1.0/config/configuration-file.html
+
+module.exports = function (config) {
+  config.set({
+    basePath: '',
+    frameworks: ['jasmine', '@angular-devkit/build-angular'],
+    plugins: [
```

Comment:
I don't think we need to specify these in here. @straker Thoughts?

Reply:
The tests are not running correctly and are running into errors, If I remove this from the karma.conf.js file.

Reply:
My guess is that since the `@angular-devkit/build/angular/plugins/karma` doesn't [start with `karma-*`](https://karma-runner.github.io/2.0/config/plugins.html) then it isn't loaded automatically. If you even have to specify a plugin to load, then you have to specify all of them.

Reply:
I did. This is how the whole piece looks like.

`  config.set({
    basePath: '',
    frameworks: ['jasmine', '@angular-devkit/build-angular'],
    plugins: [
      require('karma-jasmine'),
      require('karma-chrome-launcher'),
      require('karma-jasmine-html-reporter'),
      require('karma-coverage'),
      require('@angular-devkit/build-angular/plugins/karma'),
      require('@axe-devtools/karma-json-reporter'),
    ]`